presentation/src/main/java/com/stop/ui/map/MapFragment.kt

```diff
@@ -241,6 +251,11 @@ class MapFragment : Fragment(), MapHandler {
         }
     }
 
+    fun setMissionStart() {
+        Log.d("MissionWorker","mission 버튼 클릭")
+        binding.root.findNavController().navigate(R.id.action_mapFragment_to_missionFragment)
```

Comment:
```suggestion
       findNavController().navigate(R.id.action_mapFragment_to_missionFragment)
```
종성 님한테 리뷰 받은 부분인데, Fragment를 이용해서 navController를 찾는 `findNavController`가 view의 depth가 깊을 경우 미세하지만 더 빠르다고 합니다.

Reply:
아하 그렇군요!!